tests/test_streaming_download_manager.py

```diff
@@ -39,6 +40,9 @@
 Charmander, fire
 Squirtle, water
 Bulbasaur, grass"""
+TEST_GG_DRIVE2_ZIPPED_URL = "https://drive.google.com/uc?export=download&id=1X4jyUBBbShyCRfD-vCO1ZvfqFXP3NEeU"
+TEST_GG_DRIVE2_FILENAME = ["atoz/test1.txt", "atoz/test2.txt", "atoz/test3/test3.txt"]
+TEST_GG_DRIVE2_CONTENT = ["Erwin is the best character", "this is test 2", "foo"]
```

Comment:
For such tests I would use another host than Google Drive, which is often unreliable.
In this file you can see some tests that are using files hosted on Google Drive, but this is just to make sure that streaming from Google Drive works (it required a special logic). For the others tests let's use better hosts.

I downloaded your zip and put it here if it can be useful:
https://huggingface.co/datasets/lhoestq/test_zip_txt/resolve/main/atoz.zip